src/site/markdown/server/manage-webadmin.md

```diff
@@ -2816,6 +2818,58 @@ while:
  - userExportFrom: export deleted messages from this user
  - exportTo: content of deleted messages have been shared to this mail address
  - totalExportedMessages: number of deleted messages match with json query, then being shared to sharee
+ 
+### Purge Deleted Messages
+ 
+You can overwrite 'retentionPeriod' configuration in deletedMessageVault or use default value is 1 year.
```

Comment:
What is `deletedMessageVault` for the API user?
Be careful, people consuming the API are not the ones that manage the server. First define who you talk to, then write to her.